units/periodic_builds.go

```diff
@@ -96,7 +96,15 @@ func (j *periodicBuildJob) Run(ctx context.Context) {
 		}))
 	}()
 
-	versionID, versionError := j.addVersion(ctx, *definition)
```

Comment:
(comment placement is random)

Since this is hard to test in staging, would it be possible to come up with a plan for monitoring this after the deploy to make sure that it's working as expected?

Reply:
We'll know if this fix works if Luke's next periodic build comes up correctly.